crates/transport_tx5/src/test.rs

```diff
@@ -45,6 +48,7 @@ impl Test {
                 tx5_transport: config::Tx5TransportConfig {
                     signal_allow_plain_text: true,
                     server_url: format!("ws://127.0.0.1:{}", this.port),
+                    timeout_s: tx5_timeout_s.unwrap_or(60),
```

Comment:
```suggestion
                    timeout_s: tx5_timeout_s.unwrap_or_default(),
```

Reply:
Won't that pick the default for the type, rather than for the field? Not sure what behavior to expect there

Reply:
Yes, I think so too. It would pick the default for the type which would be `0`. Not ideal that I cannot use the actual intended default here easily though I agree...but I didn't see a better way.